versions/3.0.md

```diff
@@ -1123,6 +1125,344 @@ application/json:
   breed: Mixed
 ```
 
+#### <a name="linksObject"></a>Links Object
+The links object represents a set of possible design-time links for a response.  The presence of a link does not guarantee the caller's ability to successfully call invoke it, rather it provides a known relationship and traversal mechanism between responses and other operations.
+
+As opposed to _dynamic_ links--that is, links provided **in** the response payload, the OAS linking mechanism does not require that link information be provided in a specific response format at runtime.
+
+For computing links, and providing instructions to execute them, a mechanism is defined for accessing values in a response and using them as variables while invoking the linked operation.
+
+Field Name | Type | Description
+---|:---:|---
+<a name="referenceRef"></a>$ref | `string` | If present, a reference to another Links Object.  Note, the presence of `$ref` in the Links Object will cause all _Patterned Objects_ to be ignored. 
+
+Field Pattern | Type | Description
+---|:---:|---
+<a name="linkName"></a>link name | [Link Object](#linkObject) | A short name for the link, following the naming constraints of the <a href="#definitionsName">definitions name</a>.  The link shall reference a single Link Object, or a JSON Reference to a single link object
+
+
+### Link Object
+The `Link Object` is responsible for defining a possible operation based on a single response.
```

Comment:
Why not just defer to the web linking spec [rfc5988](https://tools.ietf.org/html/rfc5988) to define what a link looks like? Will be especially useful if we want to link to an external specification for e.g. [links in HAL](https://tools.ietf.org/html/draft-kelly-json-hal-08#section-5). The only addition to that would be an `operationId` property to dereference operations within a swagger specification.